bundle/manifests/redhat-ods-operator-manager-config_v1_configmap.yaml

```diff
@@ -1,27 +0,0 @@
-apiVersion: v1
```

Comment:
we deleted this file? :thinking:

Reply:
It's in Jira request. How is that supposed to be used? Our manager is configured with command line as a part of Deployment in CSV, isn't it?

I see mounts of `manager-config` ConfigMap in some other manifests, but not `redhat-ods-operator-manager-config`

@dchourasia